mono/tests/test-runner.cs

```diff
@@ -219,6 +238,14 @@ class TestInfo {
 							Console.Write (".");
 						passed.Add(process_data [dead]);
 						npassed ++;
+					// FIXME: is this valid for all targets?
+					} else if (dead.ExitCode == 137) {
```

Comment:
I couldn't find a list of what C# uses as ExitCodes for this sort of stuff so I just tried and saw what it did.